.github/workflows/check.yml

```diff
@@ -125,20 +125,21 @@ jobs:
         submodules: true
     - name: Create logs dir
       run: mkdir test-logs
-    - name: autogen.sh
-      run: NOCONFIGURE=1 ./autogen.sh
     - name: configure
       # We don't do gtk-doc or GObject-Introspection here, because they can
       # clash with AddressSanitizer. Instead, the clang build enables those.
       run: |
-        mkdir _build
-        pushd _build
-        ../configure  --enable-internal-checks --enable-asan --disable-introspection --without-curl
-        popd
+        meson _build \
```

Comment:
```suggestion
        meson setup _build \
```

Reply:
Hmm, or is 20.04 too old for these commands?

Reply:
Let's see what CI says.

Reply:
Turns out that Ubuntu 20.04 knows `meson setup` but not `meson compile`.